app/mod_webui/templates/base.html

```diff
@@ -36,7 +37,7 @@ <h2>Maybe all team scores?</h2>
     <footer class="page-footer">
       <div class="footer-copyright">
         <div class="container">
-          © 2017 Farooq Designs. All rights reserved.
+          © 2018 RITC3
```

Comment:
Just making sure that you checked the licensing on this and that we can change the copyright. If it's FOSS, do we have the code that says we used their template?